src/drivers/QtTestDriver.cpp

```diff
@@ -0,0 +1,31 @@
+#include "cucumber-cpp/internal/drivers/QtTestDriver.hpp"
+
+#include <QtTest/QtTest>
+#include <QTextStream>
+#include <QTemporaryFile>
+
+namespace cucumber {
+namespace internal {
+
+const InvokeResult QtTestStep::invokeStepBody() {
+    QTemporaryFile file;
+    if (!file.open()) {
+        return InvokeResult::failure("Unable to open temporary file needed for this test");
+    }
+    file.close();
+
+    QtTestObject testObject(this);
+    int returnValue = QTest::qExec(&testObject, QStringList() << "test" << "-o" << file.fileName());
+    if (returnValue == 0)
+        return InvokeResult::success();
```

Comment:
Please put curly braces around this statement (using different styles for the `if` and `else` branch is inconsistent).